### DISCUSSION TITLE
Block Element Jumps Position

### DISCUSSION THREAD

**Discussion by emrahkan** [jeffreyvr/dropblockeditor#33] [General]
Do you happen to have any ideas on how we can solve this with the editor? Whenever I edit a block element, it jumps up and doesn't stay in the same position. 😀

**Comment by jeffreyvr** [comment]
Not able to reproduce the problem you are describing. Can you provide some more information? Code/screenshots/clip can help me to better replicate your situation.

**Comment by emrahkan** [comment]
Here is an example :)

https://github.com/jeffreyvr/dropblockeditor/assets/17336614/6d2b6e19-6fe0-435f-9dd8-4abe3490279b
